packages/fxa-auth-server/lib/routes/session.js

```diff
@@ -325,15 +325,12 @@ module.exports = function (
         const sessionAal = sessionToken.authenticatorAssuranceLevel;
 
         // Build response
-        const accountEmailVerified =
-          account.emails?.primaryEmail?.isVerified || false;
+        const accountEmailVerified = account.primaryEmail.isVerified || false;
 
-        const sessionVerificationMethod = sessionToken.verificationMethod;
+        const sessionVerificationMethod = sessionToken.verificationMethodValue;
 
         // See verified-session-token auth strategy
-        const sessionVerificationSuccessful =
-          sessionToken.tokenVerificationId == null &&
-          sessionToken.tokenVerified !== false;
+        const sessionVerified = !sessionToken.tokenVerificationId;
```

Comment:
`tokenVerified` checks exactly the same thing as `!sessionToken.tokenVerificationId`, so I just removed the redundancy. I will be doing this in more places in FXA-12454.

Reply:
When I wrote this, I just reused logic that I found somewhere else. I think there's several places where this is happening.